packages/utils/src/cache/constants.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * static prefix our cache version
+ */
+const PREFIX = 'ef-cache-';
```

Comment:
```suggestion
**
 * Default cache storage name prefix
 */
export const PREFIX = 'ef-cache-';
```

Reply:
Updated by prefix value is `ef`. I and Goh propose an idea to use enum `StoragePrefix.DEFAULT`. It uses for literal type to resolve type casting issue.